Jenkinsfile

```diff
@@ -124,6 +130,8 @@ node('ubuntu-zion-legacy') {
             [scanPattern: "container:${imageName}"],
             [scanPattern: "container:${imageName}-slim"],
             [scanPattern: "container:${imageName}-redhat"],
+            [scanPattern: "container:${imageName}-alpine"],
+            [scanPattern: "container:${imageName}-alpine-slim"],
```

Comment:
I agree, but we should hold on making such changes yet. 
There is a BNR team initiative to re-write this build job completely, since it's an ad-hoc one that does no adhere to their practices. The new job will build one image at a time and it will have its own policy evaluation step targeting a single image.